opsdroid/parsers/rasanlu.py

```diff
@@ -134,7 +134,10 @@ async def _load_model(config):
             _LOGGER.error(_("Unable to connect to Rasa NLU."))
             return None
         if resp.status == 204:
-            result = await resp.json()
+            try:
+                result = await resp.json()
```

Comment:
Would it make sense to check if we have `resp.text()` if this call fails?

Reply:
Just tried it `resp.text()`. It's empty but at least it doesn't throw an exception.

Reply:
An alternative would be also to check `if resp.content_length == 0` then just `return {}` to avoid throwing and catching the exception.

Reply:
Oh right... 🤦  I just noticed that this is handling 204 which should be no content anyway haha

In this case is likely that resp will always have a length of 0.  Maybe we can just return {}. Either way this seems okay 😄